tests/slow/ParallelRestoreTestSplitMutation.txt

```diff
@@ -0,0 +1,6 @@
+testTitle=UnitTests
+testName=UnitTests
+startDelay=0
+useDB=false
+maxTestCases=0
+testsMatching=/FastRestore/RestoreLoader/splitMutation
```

Comment:
Since this is a unit test, I suggest not checking in this file, because we already have a random unit test which will cover this test.

If you do want to check in this, I'd suggest put directly under "tests" directory.

Reply:
I did this on purpose to increase the chance the test was hit by Joshua. That's why I put it here.
Later, I'm thinking about changing the way `splitMutation` is implemented and it can be still useful.

After the new backup and restore is done, we can remove this one.